Traceback (most recent call last):
  File "elysium/elysium.py", line 818, in <module>
    main()
  File "elysium/elysium.py", line 224, in main
    detected_bugs += run_osiris_bytecode_analyzer(runtime_bytecode)
  File "/Elysium/elysium/detectors/osiris.py", line 13, in run_osiris_bytecode_analyzer
    container = client.containers.run('christoftorres/osiris', "/bin/bash -c \"echo '"+bytecode+"' > bytecode.evm; python osiris/osiris.py -s bytecode.evm -glt 1800 -b --debug\"", detach=True, remove=True)
  File "/usr/local/lib/python3.8/site-packages/docker/models/containers.py", line 864, in run
    container.start()
  File "/usr/local/lib/python3.8/site-packages/docker/models/containers.py", line 405, in start
    return self.client.api.start(self.id, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/docker/utils/decorators.py", line 19, in wrapped
    return f(self, resource_id, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/docker/api/container.py", line 1125, in start
    res = self._post(url)
  File "/usr/local/lib/python3.8/site-packages/docker/utils/decorators.py", line 46, in inner
    return f(self, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/docker/api/client.py", line 233, in _post
    return self.post(url, **self._set_request_timeout(kwargs))
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 637, in post
    return self.request("POST", url, data=data, json=json, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 589, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/sessions.py", line 703, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/requests/adapters.py", line 486, in send
    resp = conn.urlopen(
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 793, in urlopen
    response = self._make_request(
  File "/usr/local/lib/python3.8/site-packages/urllib3/connectionpool.py", line 537, in _make_request
    response = conn.getresponse()
  File "/usr/local/lib/python3.8/site-packages/urllib3/connection.py", line 466, in getresponse
    httplib_response = super().getresponse()
  File "/usr/local/lib/python3.8/http/client.py", line 1348, in getresponse
    response.begin()
  File "/usr/local/lib/python3.8/http/client.py", line 316, in begin
    version, status, reason = self._read_status()
  File "/usr/local/lib/python3.8/http/client.py", line 277, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/local/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
KeyboardInterrupt
